### PR TITLE
Feat: Added Quiz-Category Page

### PR DESCRIPTION
**Pull request by coldpigli**
Added the Quiz-Category Page

image

### CODE REVIEW

File: routes/dashboard.html [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+                    </div>
+                    <div class="quiz-item cat2">
+                        <div class="quiz-meta-info txt-label children-centered">
+                            <p>Around the World</p>
+                            <p>1/10</p>
+                        </div>
+                        <div class="quiz-description children-middle">
+                            <h2 class="w-50">World Cuisine Quiz</h2>
+                            <div class="quiz-img">
+                                <img src="../assets/food-image-1-removebg-preview.png" alt="">
```
Comment (krishanu2911) [inline routes/dashboard.html]: add alt text in imge tag

File: routes/dashboard.html [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+                    </div>
+                    <div class="quiz-item cat1">
+                        <div class="quiz-meta-info txt-label children-centered">
+                            <p>Food & Beverages</p>
+                            <p>1/10</p>
+                        </div>
+                        <div class="quiz-description children-middle">
+                            <h2 class="w-50">Know your Drinks</h2>
+                            <div class="quiz-img">
+                                <img src="../assets/food-image-1-removebg-preview.png" alt="">
```
Comment (krishanu2911) [inline routes/dashboard.html]: here too

File: routes/dashboard.html [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+                <div class="new-quizzes-list flex wrap">
+                    <div class="quiz-item cat2">
+                        <div class="quiz-meta-info txt-label children-centered">
+                            <p>Food & Beverages</p>
+                            <p>1/10</p>
+                        </div>
+                        <div class="quiz-description children-middle">
+                            <h2 class="w-50">Know your Ingredient</h2>
+                            <div class="quiz-img">
+                                <img src="../assets/food-image-1-removebg-preview.png" alt="">
```
Comment (krishanu2911) [inline routes/dashboard.html]: same here

**Review by krishanu2911 — COMMENTED** [review]
Everything seems good to go. Added just few suggestions

File: routes/dashboard.html (hunk 1 above)
Comment (coldpigli) [reply routes/dashboard.html]: Oh, almost missed it. Will do. Thanks.

**Review by krishanu2911 — APPROVED** [review]

File: styles/dashboard.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+.content-area{
+    margin: 1rem;
+    gap: 2.5rem;
+    grid-area: main;
+    background-color: var(--background-dark);
+}
+
+.new-quizzes-list{
+    gap: 1.2rem;
+
```
Comment (rahulcs754) [inline styles/dashboard.css]: remove  gap

File: styles/dashboard.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    padding-top: 1rem;
+    position: sticky;
+    top: 0;
+    left: 0;
+}
+
+.sidemenu-group{
+    gap: 1.5rem;
+    color: var(--font-gray);
+
```
Comment (rahulcs754) [inline styles/dashboard.css]: remove gap

File: styles/dashboard.css [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+.user-performance .card{
+    background-color: var(--pastel-blue);
+}
+
+.user-performance .card:nth-child(2){
+    background-color: var(--pastel-red);
+}
+
+
+
```
Comment (rahulcs754) [inline styles/dashboard.css]: remove space

**Review by rahulcs754 — COMMENTED** [review]
all looked good for me but some suggestions
please remove space from CSS files

I liked your clean and created different file for pages+

File: styles/dashboard.css (hunk 4 above)
Comment (coldpigli) [reply styles/dashboard.css]: okay will do. Thanks

**Review by rahulcs754 — APPROVED** [review]

**Pull request merged**